samples/word/99-preview-apis/manage-styles.yaml

```
@@ -0,0 +1,357 @@
+id: word-manage-styles
+name: Manage styles
+description: This sample shows how to perform operations on the styles in the current document and how to add and delete custom styles.
+host: WORD
+api_set:
+    WordApi: '1.8'
+script:
+    content: |
+        $("#count").click(() => tryCatch(getCount));
+        $("#add-style").click(() => tryCatch(addStyle));
+        $("#properties").click(() => tryCatch(getProperties));
+        $("#apply-style").click(() => tryCatch(applyStyle));
+        $("#font-properties").click(() => tryCatch(setFontProperties));
+        $("#paragraph-format").click(() => tryCatch(setParagraphFormat));
+        $("#border-properties").click(() => tryCatch(setBorderProperties));
+        $("#delete-style").click(() => tryCatch(deleteStyle));
+        $("#file").change(getBase64);
+        $("#get-external-styles").click(() => tryCatch(getExternalStyles));
+
+        async function getCount() {
+          // Gets the number of styles.
```

Consider making it more evident this is about the number of styles available, not actively used.
```suggestion
          // Gets the number of styles stored with the document.
```